services/slam/fake/data_loader.go

```diff
@@ -135,6 +171,27 @@ func fakePosition(datasetDir string, slamSvc *SLAM, name string) (*referencefram
 	return pInFrame, nil
 }
 
+func fakeGetPosition(datasetDir string, slamSvc *SLAM) (spatialmath.Pose, string, error) {
```

Comment:
[q] Will GetPosition return a `spatialmath.Pose` or still a `spatialMath.PoseInFrame`? I know it changed on the endpoint, is it also changing on the interface?

Reply:
It will return a `spatialmath.Pose` Yes we are also changing the interface. This was added here: https://github.com/viamrobotics/rdk/pull/1961/files#diff-1a25b4c35c6f9f055e8d8697fd08c4006a18cc3f98514939931c458ce5edf5e5R75